qa/L0_config/config_generator.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Have you rebased? It shows these files as if they are part of the current PR.

Reply:
yes I rebased it this is quite strange